python/GafferUI/PathListingWidget.py

```diff
@@ -1082,6 +1083,8 @@ def __drawRowSelectionHighlight( self, painter, rect, index ) :
 
 			if descendantMatch and not( cellMatch & IECore.PathMatcher.Result.ExactMatch ) :
 				self.__drawHighlight( painter, cellRect, 50 )
+			elif rowMatch and not ( cellMatch & IECore.PathMatcher.Result.ExactMatch ) :
+				self.__drawHighlight( painter, cellRect, 25 )
```

Comment:
Repeating `and not ...` makes it obvious that the original code was a bit weird. I think we can move the `if cellMatch & ExactMatch` to the top, and then the subsequent tests don't need the `and not` at all?

Reply:
Tidied this up in 172b33f, does read a lot better this way round.